scripts/bump.ts

```diff
@@ -0,0 +1,18 @@
+import { exec } from "@actions/exec";
+import fs from "node:fs";
+import path from "node:path";
+import pkgJson from "../package.json" with { type: "json" };
+
+process.chdir(path.join(import.meta.dirname, ".."));
```

Comment:
We only use `import.meta.dirname` in tests and script so far so it's fine anyway. But I wonder if there are any compatibility gotchas to consider if we'd use this in the code of the action itself.

I'm not sure what is the caveat the docs mention [here](https://nodejs.org/api/esm.html#importmetadirname):
> Caveat: only present on file: modules.

Is that only that it wouldn't be present for modules imported/downloaded using HTTP(s) etc?

Reply:
Yeah I think that's what it means. On GitHub workflows the code should be ran as files though and shouldn't have this issue, though you need to account for it resolving from `dist` instead.

Reply:
> though you need to account for it resolving from dist instead.

Good point. It's not an issue right now as we don't use it in `src` (and all files are nested directly in `src` too). But I'll keep that in mind as something to be more careful about.